inborutils.Rproj

```diff
@@ -18,3 +18,5 @@ StripTrailingWhitespace: Yes
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
+PackageCheckArgs: --as-cran
+PackageRoxygenize: rd,collate,namespace
```

Comment:
@hansvancalster I don't know the precise implications of these two lines, but it is package wide and should probably  be reviewed in a separate PR?

Reply:
Sorry :wink: I just misused your PR to add these lines to the .Rproj file. They aid in checking the package when it is built. @ThierryO advised to include these settings in the .Rproj file and he can tell you more.

Reply:
You're welcome.